database/buildSchema/43_views_functions_triggers.sql

```diff
@@ -162,10 +162,6 @@ CREATE TRIGGER trigger_queue
 
 DROP FUNCTION IF EXISTS public.set_template_verision CASCADE; -- DEPRECATED
 
--- Enforce unique versionId per template type
-ALTER TABLE public.template
-    ADD UNIQUE (code, version_id);
-
```

Comment:
This is what caused the duplicate template indexes. Doesn't need to be in here, just added the corrected version of this to the migration script instead.